README.md

```diff
@@ -20,13 +20,15 @@ yarn build
 yarn test
 ```
 
-### Deploy
+### Deployments
+
+A collection of the different Safe contract deployments and their addresses can be found in the [Safe deployments](https://github.com/gnosis/safe-deployments) repository.
 
-Some contracts require that the Solidity compile target is at least `petersburg` (e.g. GnosisSafeProxyFactory and MultiSend). This is default since [Solidity 0.5.5](https://github.com/ethereum/solidity/releases/tag/v0.5.5).
+To add support for a new network follow the steps of the ``Deploy`` section and create a PR in the [Safe deployments](https://github.com/gnosis/safe-deployments) repository. 
 
-Note: The formal verification was performed using the contract compiled with solcjs 0.5.0.
+### Deploy
 
-This will deploy the contracts deterministically and verify the contracts on etherscan.
+This will deploy the contracts deterministically and verify the contracts on etherscan using [Solidity 0.7.6](https://github.com/ethereum/solidity/releases/tag/v0.7.6) by default.
```

Comment:
Not sourcify?

Reply:
etherscan is default, sourcify is not stable in some cases (e.g. when using immutableS)